lib/teleterm/apiserver/handler/handler_access_requests.go

```diff
@@ -0,0 +1,155 @@
+// Copyright 2021 Gravitational, Inc
```

Comment:
FWIW, in the future you can run `make fix-license` and it'll automatically add the license to files but it'll have the current year. ;)